code.md

```diff
@@ -31,7 +31,7 @@ If a participant engages in harassing behaviour, event organisers retain the rig
 
 Event organisers may take action to redress anything designed to, or with the clear impact of, disrupting the event or making the environment hostile for any participants.
 
-We expect participants to follow these rules at all event venues and event-related social activities. We think people should follow these rules outside event activities too!
+We expect participants to follow these rules at all event venues and event-related social activities, and will strongly consider forums where conference attendees interact outside the conference (including social media, personal emails, phone calls or texts, or in-person interaction) to be held to the same code of conduct. Please behave professionally.
```

Comment:
"We expect ... and will strong consider..." doesn't parse well for me. I'm left unclear if the CoC will enforce breaches on social media / email etc or not. It would be stronger language, but how about "We require ... and expect..." here?

Reply:
Yeah, we want to remove any grey area at all. "require" and "expect" for me too.

Reply:
good points.